commands/import.go

```diff
@@ -72,7 +72,13 @@ func (i *Importer) parseArgs() error {
 		return cli.NewExitError("\n\t[!] Both <directory to import> and <database name> are required.", -1)
 	}
 
-	err := i.checkForInvalidDBChars(i.targetDatabase)
+	// check if import directory is okay to read from
+	err := i.checkImportDirExists()
+	if err != nil {
+		return err
```

Comment:
Should this return a `cli.NewExitError` like the other checks in this function?

Reply:
checkImportDirExists returns a cli exit error. The wrapping call transforming the normal error into a cli exit error could be moved here however.